target/linux/ath79/dts/qca9531_engenius_ews511ap.dts

```diff
@@ -0,0 +1,143 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	compatible = "engenius,ews511ap", "qca,qca9531";
+	model = "EnGenius EWS511AP";
+
+	keys {
+		compatible = "gpio-keys-polled";
+
+		poll-interval = <20>;
+		pinctrl-names = "default";
+		pinctrl-0 = <&jtag_disable_pins>;
+
+		button0 {
+			label = "reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 1 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		power {
+			label = "ews511ap:amber:power";
+			gpios = <&gpio 0 GPIO_ACTIVE_LOW>;
+			default-state = "on";
+		};
+
+		lan1 {
+			label = "ews511ap:blue:lan1";
+			gpios = <&gpio 4 GPIO_ACTIVE_LOW>;
+			default-state = "off";
```

Comment:
Please drop this `default-state = "off"` property for details see https://openwrt.org/submitting-patches#dts_checklist

Reply:
Done. Thanks for your reminder.